docs/code-snippets/outlook-snippets.yaml

```diff
@@ -1346,10 +1346,18 @@ Office.Recurrence.getAsync:
     function callback(asyncResult){
         var context = asyncResult.context;
         var recurrence = asyncResult.value;
-        var endDate = recurrence.seriesTime.getEndDate();
+        if (recurrence == null) {
```

Comment:
The only thing I can think of here that might be weird is the ==? I'm not sure if recurrence ever ends up being a value that makes this == do weird things, but it might be worth making into a === to be super sure that there is strict equivalency to null? Not sure if we have any guidance in terms of doing === vs. ==. @kbrandl, @Rick-Kirkham, @davidchesnut do you know if we prefer one over the other? Thanks